ngraph_bridge/ngraph_builder.cc

```diff
@@ -5056,6 +5136,27 @@ Status Builder::TranslateGraph(
     result->set_needs_default_layout(true);
   }
 
+  // TODO: Use is_type<ng::op::Result>(n) when we upgrade to new ngraph
```

Comment:
Reviewed this part. We are safeguarded by the check, if some node is not tagged we will know.

Reply:
Is nGraph is only attaching the provenance tags when certain env flags are enabled then we might run

Reply:
Its always going to attach tags in builders. If flags are present it will start to do something with the tags.